src/main/java/uk/gov/hmcts/cmc/claimstore/services/ClaimService.java

```diff
@@ -354,4 +355,13 @@ public void saveReDetermination(
     public void updateClaimState(String authorisation, Claim claim, ClaimState state) {
         caseRepository.updateClaimState(authorisation, claim.getId(), state);
     }
+
+    public void updateClaimSubmissionOperationIndicators(String authorisation,
```

Comment:
i see different formatting for this every time 👀 maybe for now we should just be consistent with rest of class and refactor everything together

Reply:
@markry-kainos @srineelesh pls let we not refactor rest of class for this change. I think it should be taken as separate PR. Also we havent as team agreed what is standard yet. So just leave as it is

Reply:
@markry-kainos I have changed the format to keep it consistent with the others. I personally don't prefer this format because the code format is in accordance with checkstyle and intellij. I haven't changed the entire class because as Karthik mentioned I too feel it should be in a separate PR. I think we should decide as a team what format to follow and best is to set it up in cheekstyle.

Reply:
cool - i meant that - keep same and we change big later 😊 👍